src/pystack/_pystack/process.cpp

```diff
@@ -535,6 +535,20 @@ AbstractProcessManager::Pid() const
 {
     return d_pid;
 }
+
+remote_addr_t
+AbstractProcessManager::getAddressFromCache(const std::string& symbol) const
+{
+    return d_symbol_cache[symbol];
```

Comment:
Er, I think you meant to do
```suggestion
    return d_type_cache[symbol];
```